package.json

```diff
@@ -38,5 +38,6 @@
     "ts-jest": "29.1.2",
     "typescript": "5.4.5",
     "typescript-eslint": "7.8.0"
-  }
+  },
+  "packageManager": "pnpm@9.4.0+sha512.f549b8a52c9d2b8536762f99c0722205efc5af913e77835dbccc3b0b0b2ca9e7dc8022b78062c17291c48e88749c70ce88eb5a74f1fa8c4bf5e18bb46c8bd83a"
```

Comment:
What's this auto TS generate commit?

Reply:
It's from this workflow: https://github.com/iFixit/datadog-actions-metrics-2/actions/runs/9749642921/job/26907275674#step:9:108